### PR TITLE
Added FISh docs

### LINKED ISSUE #103: Document FISh in README

### PR DESCRIPTION
**Pull request by MyIgel**
Very FIShy, much wow

Closes #103 (Document FISh in README)

### CODE REVIEW

**Comment by vifino** [comment]
There is also no `/` stuff: `modname args` calls module *now*, interrupting the current one.

File: README.md [hunk 1]
```diff
@@ -261,6 +261,24 @@ If you want to build a specific module later on, you can do the same but with ju
...
+| --- | --- |
+| `/blank` | Blank the output |
+| `/error42` | Exit the program |
+| `/next` | Jump to the next effect |
+| `/then <modulename> [parameters]` | Schedule the given module or command with optional parameters |
+
+Examples:
+```bash
+echo '/next' > sled.fish
+echo '/then /blank' > sled.fish
```
Comment (basxto) [inline README.md]: Make clear that '/then autoterminal /next' is not possible and '/then autoterminal\n/next' is neccessary

Comment (MyIgel) [reply README.md]: Funnyly it crashes when using `echo -e "/then text Ipsum\n/next" > sled.fish`...

File: README.md [hunk 2]
```diff
@@ -261,6 +261,24 @@ If you want to build a specific module later on, you can do the same but with ju
...
 Or, if you are particularly lazy, just add the module name to the make invocation.
 
+## FISh
+The `sled.fish` FiFo supports the following commands:
+
+| Command | Action |
+| --- | --- |
+| `/blank` | Blank the output |
+| `/error42` | Exit the program |
+| `/next` | Jump to the next effect |
```
Comment (vifino) [inline README.md]: *Technically*, there is no `/next`. It's treated the same as any other unknown command.

Comment (MyIgel) [reply README.md]: I know, but thats imho not important here (and a practical example)?

**Comment by MyIgel** [comment]
Funfact: The `Misaligned table fence` Codacy error is totally crap.....
Why on f[beep] earth do i even want to have to change all lines in a table if i only change the longest one??

File: README.md (hunk 1 above)
Comment (MyIgel) [reply README.md]: (at least that got fixed in #108 (text: Fix use after free in text_free)

**Pull request merged**